src/generators/operationGenerator.ts

```diff
@@ -1119,7 +1233,9 @@ export function addOperationSpecs(
         declarations: [
           {
             name: operationSpec.name,
-            type: "coreHttp.OperationSpec",
+            type: !optionsBag.useCoreV2
+              ? "coreHttp.OperationSpec"
```

Comment:
I wonder if we can have named import for OperationSpec? it may reduce the number of checks for useCoreV2 considerably

Reply:
I think this one we can keep it as such. The main reason is we have dependencies such as OperationSpec, OperationOptions, RequestOptionsBase, operationOptionsToRequestOptionsBase, OperationArguments, etc.